projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -69,7 +69,11 @@ function process_anchor_params() {
 
 	$current_screen = \get_current_screen();
 	// TODO: Replace `$current_screen->is_block_editor()` with `wp_should_load_block_editor_scripts_and_styles()` that is introduced in WP 5.6.
-	if ( method_exists( $current_screen, 'is_block_editor' ) && ! $current_screen->is_block_editor() ) {
+	if (
+		is_object( $current_screen )
+		&& method_exists( $current_screen, 'is_block_editor' )
```

Comment:
A shorter alternative would be
```suggestion
		is_callable( array( $current_screen, 'is_block_editor' ) )
```
Or, since it seems that `get_current_screen()` only returns a screen object or null and the screen object has had `->is_block_editor()` since WP 5.0, it should be sufficient to just do
```suggestion
		$current_screen instanceof \WP_Screen
```
or even
```suggestion
		$current_screen
```
assuming the WordPress ecosystem isn't in the habit of ignoring the `@return WP_Screen|null` in favor of shoving other things into the global.

Reply:
Good call. I went with your second suggestion in 800fea1f961737c9797320eebbff10f1c6aa609a. Option 3 seems a bit risky, who knows what other plugins do in the wild.